src/Network/IMAP/Parsers/Untagged.hs

```diff
@@ -161,7 +161,7 @@ parseExpunge = do
 
 parseSearchResult :: Parser (Either ErrorMessage UntaggedResult)
 parseSearchResult = do
-  string "SEARCH "
-  msgIds <- AP.takeWhile1 isDigit `sepBy` char ' '
-  let parsedIds = mapM toInt msgIds
+  string "SEARCH"
+  msgIds <- AP.takeWhile isDigit `sepBy` char ' '
```

Comment:
is there a case when this caused a bug?

Reply:
I believe if there are no search results the original fails.

Reply:
ah, good catch, thanks :)